pkg/processor/trigger/trigger.go

```diff
@@ -95,11 +92,8 @@ type Trigger interface {
 	// SignalWorkersToTerminate signal to all workers that the processor is about to stop working
 	SignalWorkersToTerminate() error
 
-	// PreBatchHooks does trigger-specific actions before sending a batch
-	PreBatchHooks(batch []nuclio.Event, workerInstance eventprocessor.EventProcessor)
-
-	// PostBatchHooks does trigger-specific actions after sending a batch
-	PostBatchHooks(batch []nuclio.Event, workerInstance eventprocessor.EventProcessor)
```

Comment:
U Sure these will not be needed at all?

Reply:
@TomerShor this was needed only for the purpose of updating http-specific values, so now I don't see any reason why would we need it in the future. But if we will, can be added